src/improvements/justfile

```diff
@@ -143,4 +144,35 @@ simulate-verify-task TASK="":
     # For testing purposes, we do not gain anything by simulating as other nested safes.
     nested_safe_name="foundation"
 
-    ${root_dir}/src/improvements/script/simulate-verify-task.sh {{TASK}} $nested_safe_name
+    # If TASK doesn't start with / make it absolute using $PWD
+    if [[ "{{TASK}}" != /* ]]; then
```

Comment:
So is the user expected to pass through an absolute path here?

Reply:
Also, where is the `simulate-verify-task TASK="":` just command expected to be executed from?

Reply:
I added comments about when the command is supposed to be used, and what the parameter is